### PR TITLE
Use the <meta name="generator"> tag for dartdoc version info

### PR DESCRIPTION
**Pull request by kevmoo**
Closes https://github.com/dart-lang/dartdoc/issues/999

CC @sethladd

[Review on Reviewable](https://reviewable.io/reviews/dart-lang/dartdoc/1018)

### CODE REVIEW

File: lib/templates/_head.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="generator" content="dartdoc {{version}}">
```
Comment (sethladd) [inline lib/templates/_head.html]: Can you add the "Made with <3" back? I like the friendly tone.

Otherwise, thanks for the upgrade to generator!

Comment (kevmoo) [inline lib/templates/_head.html]: I love the friendly tone, too – but putting `>` (or even `<`) in tag attributes is a no-no.

...and `&gt;3` doesn't have the same feel :smile:

Comment (sethladd) [inline lib/templates/_head.html]: "made with love" ?

Comment (kevmoo) [inline lib/templates/_head.html]: I mean...I *guess* :wink:

**Comment by kevmoo** [comment]
@sethladd PTAL

**Comment by sethladd** [comment]
:)

Thanks! and lgtm

**Pull request merged**